modules/homebrew.nix

```diff
@@ -1,219 +1,751 @@
+        '';
+      };
+      language = mkNullOrStrOption {
+        description = ''
+          Comma-separated list of language codes to prefer for cask installation. The first matching
+          language is used, otherwise it reverts to the cask’s default language. The default value
+          is the language of your system.
+        '';
+        example = "zh-TW";
+      };
+      require_sha = mkNullOrBoolOption {
+        description = ''
+          Whether to require cask(s) to have a checksum.
+
+          Homebrew's default is <literal>false</literal>.
+        '';
+      };
+      no_quarantine = mkNullOrBoolOption {
+        description = "Whether to disable quarantining of downloads.";
+      };
+      no_binaries = mkNullOrBoolOption {
+        description = "Whether to disable linking of helper executables.";
+      };
+
+      brewfileLine = mkInternalOption { type = types.nullOr types.str; };
+    };
+
+    config =
+      let
+        sCfg = mkProcessedSubmodConfig config;
+      in
+      {
+        brewfileLine =
+          if sCfg == { } then null
+          else "cask_args ${mkBrewfileLineOptionsListString sCfg}";
+      };
+  };
+
+  brewOptions = { config, ... }: {
+    options = {
+      name = mkOption {
+        type = types.str;
+        description = "The name of the formula to install.";
+      };
+      args = mkOption {
+        type = with types; nullOr (listOf str);
+        default = null;
+        description = ''
+          Arguments flags to pass to <command>brew install</command>. Values should not include the
+          leading <literal>"--"</literal>.
+        '';
+      };
+      conflicts_with = mkOption {
+        type = with types; nullOr (listOf str);
+        default = null;
+        description = ''
+          List of formulae that should be unlinked and their services stopped (if they are
+          installed).
+        '';
+      };
+      restart_service = mkOption {
+        type = with types; nullOr (either bool (enum [ "changed" ]));
+        default = null;
+        description = ''
+          Whether to run <command>brew services restart</command> for the formula and register it to
+          launch at login (or boot). If set to <literal>"changed"</literal>, the service will only
+          be restarted on version changes.
+
+          Homebrew's default is <literal>false</literal>.
+        '';
+      };
+      start_service = mkNullOrBoolOption {
+        description = ''
+          Whether to run <command>brew services start</command> for the formula and register it to
+          launch at login (or boot).
+
+          Homebrew's default is <literal>false</literal>.
+        '';
+      };
+      link = mkNullOrBoolOption {
+        description = ''
+          Whether to link the formula to the Homebrew prefix. When this option is
+          <literal>null</literal>, Homebrew will use it's default behavior which is to link the
```

Comment:
```suggestion
          <literal>null</literal>, Homebrew will use its default behavior which is to link the
```

Reply:
This is already fixed in the final diff.